src/blog/2023/06/node-explained-filter.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Nodes explained: Filter"
+subtitle: The filter node is incredibly versatile, let's go over some use-cases
+description: Forwarding or blocking messages based on values, change of them, or lack of change.
+date: 2023-06-15
+authors: ["zeger-jan-van-de-weg"]
+image:
```

Comment:
Not sure if this is worth the time for an art request? Or if a generated image is fine?

Reply:
I agree it is not worth it.